lib/flatpak-ref.h

```diff
@@ -71,4 +71,8 @@ FLATPAK_EXTERN char *        flatpak_ref_format_ref (FlatpakRef *self);
 FLATPAK_EXTERN FlatpakRef *   flatpak_ref_parse (const char *ref,
                                                  GError    **error);
 
+#ifdef FLATPAK_ENABLE_P2P
+FLATPAK_EXTERN const char *   flatpak_ref_get_collection_id (FlatpakRef *self);
+#endif  /* FLATPAK_ENABLE_P2P */
```

Comment:
Also, if we’re making the getter function unconditionally public, I assume you also want the `FlatpakRef::collection-id` property to be unconditionally public?